src/state-prototypes.js

```diff
@@ -224,6 +229,7 @@ function blankState() {
     autoSave: undefined,
     locallySaved: [],
     savedEnvironment: {},
+    runningCellID: undefined,
```

Comment:
hmm does starting as `undefined` pass the schema validator requiring "{ type: 'integer' }"? I guess so if it's working... (i thought i'd encountered errors around this kind of thing before)

Reply:
I'm noticing that it does in fact pass the schema validator. Not sure why or how, but I think `undefined` is apparently valid in other places.